app/models/concerns/orchestration.rb

```diff
@@ -82,14 +89,22 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
+    @skip_orchestration = true
+  end
+
+  def skip_orchestration? # replacing skip_orchestration
```

Comment:
actually the comment was a message for you, I wanted to point out the `?` symbol :-) I think you can replace it now